Source/Controllers/MainViewControllers/TableStructure/SPTableStructure.m

```diff
@@ -2718,6 +2770,50 @@ + (SPFieldTypeHelp *)helpForFieldType:(NSString *)typeName
 	return nil;
 }
 
+- (IBAction)filterChanged:(NSSearchField *)sender
+{
+	if (sender == filterSearchField && [self filterFieldsWithString:sender.stringValue])
+		[tableSourceView reloadData];
+}
+
+- (BOOL)filterFieldsWithString:(NSString *)filterString
+{
+	if (selectedTable)
+	{
```

Comment:
Ideally use brackets on the same line always (I see in this PR it's not that case), but it's a nit pick

Reply:
I like this nit pick. I also prefer them on the same line but in objc I did put them on their own line to match the style already used in the existing code. Would you still like me to move them up and get mixed style in the same file?

Reply:
I would move them up like @Kaspik said! Yeah, it will make the file mixed but it's already sorta mixed to begin with so I'm not too worried about it. Realistically we should set up some automated linting! :)

Reply:
Some linting would be nice! Moved them up with latest commit. Marking this one as resolved.